analysis_options.yaml

```diff
@@ -18,6 +18,12 @@ analyzer:
   exclude: # DIFFERENT FROM FLUTTER/FLUTTER
     # Fixture depends on dart:ui and raises false positives.
     - flutter_frontend_server/test/fixtures/lib/main.dart
+    - examples/
+    - impeller/
+    - prebuilts/
+    - runtime/
+    - shell/
```

Comment:
Why are these excluded?

Reply:
Shoot, this leaked in from another PR (the multi-pub one). Reverting.

Reply:
(I actually think it makes sense to exclude them still, but let's do it in a standalone PR so I can explain why)